core-bundle/src/Resources/contao/modules/ModuleRegistration.php

```diff
@@ -455,14 +455,20 @@ protected function createNewUser($arrData)
 	/**
 	 * Send the activation mail
 	 *
-	 * @param array $arrData
+	 * @param integer     $intId
+	 * @param MemberModel $objNewUser
 	 */
-	protected function sendActivationMail($arrData)
+	protected function sendActivationMail($intId, $arrData)
```

Comment:
BC break.

Reply:
I will update the method signature and make the second argument optional. But isn't this a BC break anyways? The token service needs the ID of the new member and `$arrData` does not contain it.